### PR TITLE
fix PerfMetrics

### PR DESCRIPTION
**Pull request by pavel-esir**
CVS-151497

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

File: src/cpp/include/openvino/genai/perf_metrics.hpp [hunk 1]
```diff
@@ -111,8 +108,8 @@ struct OPENVINO_GENAI_EXPORTS PerfMetrics {
     size_t num_input_tokens;
     
     float get_load_time();         // Load time in ms.
-    float get_num_generated_tokens();
-    float get_num_input_tokens();
+    size_t get_num_generated_tokens();
+    size_t get_num_input_tokens();
     MeanStdPair get_ttft();         // Time to the first token (in ms) (TTTFT).
```
Comment (apaniukov) [inline src/cpp/include/openvino/genai/perf_metrics.hpp:113]:
```suggestion
    MeanStdPair get_ttft();         // Time to the first token (in ms) (TTFT).
```

**Review by Wovchena — APPROVED** [review]
Great idea measuring time once more using python!
Will you disable broken windows tests? Example for master: https://github.com/openvinotoolkit/openvino.genai/pull/807

**Comment by Wovchena** [comment]
We would also need sample tests

**Comment by Wovchena** [comment]
And port to master please

**Pull request merged**